website/static/css/registration_landing.css

```diff
@@ -0,0 +1,146 @@
+/* PREREG PAGES */
+
+body, html {
+    height: 100%;
+}
+
+.reg-landing-page-container ol, .reg-landing-page-container p,  .reg-landing-page-button {
```

Comment:
There's a lot of duplication with the prereg onboarder CSS. Can you refactor so that the shared styles are in a CSS file used on both pages?